Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1717,6 +1710,43 @@ static bool isContextMenuEvent(const WebMouseEvent& event)
         scrollToPDFDestination(destination);
 }
 
+WebCore::FloatRect UnifiedPDFPlugin::contentsRectForAnnotation(RetainPtr<PDFAnnotation> annotation) const
```

Comment:
Why is this taking a `RetainPtr<>`? The implementation doesn't `WTFMove()` it so there is no benefit. I think this should be a raw pointer.